README.md

```diff
@@ -295,8 +295,9 @@ These are the available config options for the middleware/plugin functions. All
   // Used if useHeader/echoHeader is set to true.
   headerName: 'X-Request-Id',
   // A custom function to generate your request ids (default: UUID v1).
+  // Optionally, request could be used to calculate the id.
```

Comment:
It's better to be more precise and use smth like `Framework-specific request object is passed as the argument, so it could be optionally used to calculate the id.`